File: examples-sdk/python/038-basic-vil-app-dsl.py
#!/usr/bin/env python3
"""038-basic-vil-app-dsl — Python SDK equivalent
Compile: vil compile --from python --input 038-basic-vil-app-dsl.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()

File: examples-sdk/typescript/038-basic-vil-app-dsl.ts
#!/usr/bin/env tsx
// 038-basic-vil-app-dsl — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 038-basic-vil-app-dsl.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();
